tests/unit/test_cli.py

```diff
@@ -526,102 +581,100 @@ def test_migrate_locations_aws(ws, caplog):
+def test_create_catalogs_schemas(ws1):
     prompts = MockPrompts({'.*': 's3://test'})
-    ws.external_locations.list.return_value = [ExternalLocationInfo(url="s3://test")]
-    create_catalogs_schemas(ws, prompts)
-    ws.catalogs.list.assert_called_once()
+    ws1.external_locations.list.return_value = [ExternalLocationInfo(url="s3://test")]
+    create_catalogs_schemas(ws1, prompts)
+    ws1.catalogs.list.assert_called_once()
 
 
-def test_create_catalogs_schemas_handles_existing(ws, caplog):
+def test_create_catalogs_schemas_handles_existing(ws1, caplog):
     prompts = MockPrompts({'.*': 's3://test'})
-    ws.external_locations.list.return_value = [ExternalLocationInfo(url="s3://test")]
-    ws.catalogs.create.side_effect = [BadRequest("Catalog 'test' already exists")]
-    ws.schemas.create.side_effect = [BadRequest("Schema 'test' already exists")]
-    create_catalogs_schemas(ws, prompts)
-    ws.catalogs.list.assert_called_once()
+    ws1.external_locations.list.return_value = [ExternalLocationInfo(url="s3://test")]
+    ws1.catalogs.create.side_effect = [BadRequest("Catalog 'test' already exists")]
+    ws1.schemas.create.side_effect = [BadRequest("Schema 'test' already exists")]
+    create_catalogs_schemas(ws1, prompts)
+    ws1.catalogs.list.assert_called_once()
 
     assert "Catalog test already exists. Skipping." in caplog.messages
     assert "Schema test in catalog test already exists. Skipping." in caplog.messages
 
 
-def test_cluster_remap(ws, caplog):
+def test_cluster_remap(ws1, caplog):
     prompts = MockPrompts({"Please provide the cluster id's as comma separated value from the above list.*": "1"})
-    ws = create_autospec(WorkspaceClient)
-    ws.clusters.get.return_value = ClusterDetails(cluster_id="123", cluster_name="test_cluster")
-    ws.clusters.list.return_value = [
+    ws1.clusters.get.return_value = ClusterDetails(cluster_id="123", cluster_name="test_cluster")
+    ws1.clusters.list.return_value = [
         ClusterDetails(cluster_id="123", cluster_name="test_cluster", cluster_source=ClusterSource.UI),
         ClusterDetails(cluster_id="1234", cluster_name="test_cluster1", cluster_source=ClusterSource.JOB),
     ]
-    cluster_remap(ws, prompts)
+    cluster_remap(ws1, prompts)
     assert "Remapping the Clusters to UC" in caplog.messages
 
 
-def test_cluster_remap_error(ws, caplog):
+def test_cluster_remap_error(ws1, caplog):
     prompts = MockPrompts({"Please provide the cluster id's as comma separated value from the above list.*": "1"})
-    ws = create_autospec(WorkspaceClient)
-    ws.clusters.list.return_value = []
-    cluster_remap(ws, prompts)
+    ws1.clusters.list.return_value = []
+    cluster_remap(ws1, prompts)
     assert "No cluster information present in the workspace" in caplog.messages
 
 
-def test_revert_cluster_remap(ws, caplog, mocker):
+def test_revert_cluster_remap(caplog):
+    # TODO: What is this test supposed to test? Why do we expect a TypeError below?
```

Comment:
@prajin-29: Do you remember what this function is supposed to test? It is not clear to me why the type error below is expected given the test setup